src/commands/Admin/purge.ts

```diff
@@ -15,6 +15,8 @@ module.exports = class PurgeCommand extends Command {
       name: 'purge',
       description: 'Purges a specific number of messages',
       preconditions: ['GuildOnly', ['AdminOnly', 'BotOwnerOnly'], 'AdminPerms'],
+      detailedDescription: `${surroundStringWithBackTick(`Usage: purge <number>`)}. 
+The number represents the number of messages to purge. Maximum: 100`,
```

Comment:
Any way to append the constant into this string?

Reply:
What I'll do is make the entire "Usage: $" portion be added by the help command rather than from here. We just need to enforce that every detailed description starts like this.